config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root to: "users#index"
+  get "users/:id", to: "users#show", as:"user"
+  get "users/:author_id/posts", to: "posts#index", as: "user_posts"
+  get "users/:author_id/posts/:id", to: "posts#show", as: "user_post"
```

Comment:
## Recommended Changes

- [ ] You could set your routes in another way like this:
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```